Source/Scene/Model.js

```diff
@@ -1186,8 +1187,16 @@ define([
         var cacheKey = defaultValue(options.cacheKey, getAbsoluteUri(url));
 
         options = clone(options);
-        options.basePath = getBaseUri(url, true);
-        options.cacheKey = cacheKey;
+        if (defined(options.basePath)) {
```

Comment:
This can still be simplified a bit further
``` javascript
options.cacheKey = cacheKey;
if (defined(options.basePath)) {
    options.cacheKey += options.basePath;
} else {
    options.basePath = getBaseUri(url, true);
}
```